man/docker-create.1.md

```diff
@@ -240,6 +242,22 @@ This value should always larger than **-m**, so you should always use this with
 **-w**, **--workdir**=""
    Working directory inside the container
 
+# EXAMPLES
+
+## Running a container with --interactive and without attaching to STDIN
```

Comment:
I was under the wrong impression that there might be a strange use-case on your end, since that scenario was possible.  If not, I vote to close this PR in favor of https://github.com/docker/docker/pull/14417 !